files/scripts/supervisor-proc-exit-listener

```diff
@@ -35,12 +52,21 @@ def main():
             processname = payload_headers['processname']
             groupname = payload_headers['groupname']
 
-            # If a critical process exited unexpectedly, terminate supervisor
-            if expected == 0 and processname in critical_processes or groupname in critical_processes:
-                MSG_FORMAT_STR = "Process {} exited unxepectedly. Terminating supervisor..."
-                msg = MSG_FORMAT_STR.format(payload_headers['processname'])
-                syslog.syslog(syslog.LOG_INFO, msg)
-                os.kill(os.getppid(), signal.SIGTERM)
+            config_db = swsssdk.ConfigDBConnector()
+            config_db.connect()
+            docker_config = config_db.get_table(CONTAINER_FEATURE_TABLE_NAME)
```

Comment:
Please change `docker_config` to `container_features_table`

Reply:
Fixed.